rust/src/api.rs

```diff
@@ -184,6 +188,7 @@ pub async fn get_offer() -> Result<Option<Offer>> {
 
 #[tokio::main(flavor = "current_thread")]
 pub async fn call_faucet(address: String) -> Result<String> {
+    anyhow::ensure!(!address.is_empty(), "received empty address");
```

Comment:
🙃 Might be hard to tell this one apart from the previous one if we use the same error message.

I like the pattern that you used for the last error: `Cannot do X because of empty Y`.

Reply:
I believe they have the line number attached, but I concur that it will be easier to distinguish if I added a bit more info. added :)